code/graphics/opengl/gropenglshader.cpp

```diff
@@ -582,6 +582,13 @@ int opengl_compile_shader(shader_type sdr, uint flags)
 				glBindAttribLocation(program->getShaderHandle(), i, GL_vertex_attrib_info[i].name.c_str());
 			}
 
+			// bind fragment data locations before we link the shader
+			glBindFragDataLocation(new_shader.program->getShaderHandle(), 0, "fragOut0");
```

Comment:
`new_shader.program` isn't valid at this point so this causes a segfault at startup. Instead you must use `program`.